saul-core/doc/WORKFLOW.md

```diff
@@ -0,0 +1,39 @@
+
+A Saul programmer needs to think about doing this workflow.
+
+1) Prepare a Reader program can be written in Java or Scala to read the data in some class of objects.
```

Comment:
"Class" is confusing with Java-classes. Maybe paraphrase "in some class of objects", or drop it?